src/Photos.vue

```diff
@@ -160,6 +168,9 @@ export default {
 			showLocationMenuEntry: getCurrentUser() === null
 				? false
 				: getCurrentUser().isAdmin || isMapsInstalled,
+			showPeopleMenuEntry: getCurrentUser() === null
+				? false
+				: getCurrentUser().isAdmin || isRecognizeInstalled,
```

Comment:
No?

```suggestion
			showPeopleMenuEntry: getCurrentUser() === null
				? false
				: getCurrentUser().isAdmin && isRecognizeInstalled,

Reply:
heh, I thought the same thing, but I think it's correct like this. If the current user is admin we want to always show the entry, by default we only show it if recognize is installed. We show it always for admins because we link to the app store in case it's not installed.

Reply:
Oh, yes, of course :)

Maybe more explicit like that then:

```suggestion
			showPeopleMenuEntry: isRecognizeInstalled || getCurrentUser()?.isAdmin === true,
```

To test ;)

Reply:
Ah, but if the user is not not set, we can't use the faces feature either

Reply:
because it's per user